src/style.css

```
@@ -1 +1,63 @@
-/* add styles here */
+* {
+  margin: 0;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 22px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  flex-shrink: 0;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 > .stars__star:nth-child(0) {
```

```suggestion
.stars--1 > .stars__star:nth-child(0),
.stars--2 > .stars__star:nth-child(-n + 2),
... {
```
You can use commas to group selectors